networking/tcp/tcp_keepalive/runtest.sh

```diff
@@ -40,6 +40,14 @@ fi
 
 rlJournalStart
     rlPhaseStartSetup
+       if [ $(rlGetDistroRelease) -ge "7" ]; then
+           rlServiceStop firewalld 0-255
+       else
+           rlServiceStop iptables 0-255
+           rlServiceStop ip6tables 0-255
```

Comment:
`rlServiceStop` interprets _all_ parameters as names of services to be stopped (see https://github.com/beakerlib/beakerlib/wiki/man#rlservicestop). Can you remove the `0-255` from the calls?

Reply:
Ah, and can you please update the branch, too?

Reply:
Please see the new patch, I removed 0-255, Thanks